test/inductor/test_triton_kernels.py

```diff
@@ -33,9 +33,14 @@
 
 
 # Define shared triton constants here.
-CONSTANT_C = 4
-STRING_CONSTANT_C = "CONSTANT_C"
-BOOL_CONSTANT_C = True
+if HAS_CUDA:
```

Comment:
Previously `STRING_CONSTANT_C` was not constexpr so the test had to take the `.value` of the constexpr `CONSTANT_NAME` argument (actually, I wonder if this would have just worked b/c the LHS side is constexpr, so you get a constexpr `__eq__` attribute, which should work with a non-constexpr RHS?)

Regardless, now that the test has changed should we use the `constexpr` variant of the globals uniformly?

Reply:
I think we can just remove the `else` block, since if triton isn't available we don't run the test.